docs/get-started/getting-started-workload-qualification.md

```diff
@@ -188,7 +188,20 @@ which is the same as the driver logs with `spark.rapids.sql.explain=all`.
 
 4. Run the function `explainPotentialGpuPlan` on the query DataFrame.
 
-   For example:
+   Scala API:
+   ```scala
+   val output=com.nvidia.spark.rapids.ExplainPlan.explainPotentialGpuPlan(df)
+   ```
+
+   Python API:
+   ```python
+   output = sparkSession._jvm.com.nvidia.spark.rapids.ExplainPlan.explainPotentialGpuPlan(df._jdf, "ALL")
+   ```
+     - First parameter, `df` is the Spark DataFrame to get the query plan from.
+     - Second parameter, `ALL` returns all the explain data, you can change to `NOT_ON_GPU` to return just what
+       does not work on the GPU. The Scala API, ALL is the default when second parameter is not specified.
```

Comment:
"The Scala API," is a bit confusing here, was it intended to say "For the Scala API," or was this supposed to be in a separate section like "Python API:" above?

Reply:
really I just want the java doc of this function which we don't publish so I changed this to just have a function documentation section